cadence/contracts/ISPOManager.cdc

```diff
@@ -353,9 +356,15 @@ pub contract ISPOManager {
         }
 
         pub fun withdrawAdminFlowRewards(): @FungibleToken.Vault {
+            if self.stakingRewardsVault == nil {
```

Comment:
Is this necessary? Check line 369 it imho handles that logic

Reply:
handles the edge case I mentioned below: https://github.com/nufi-official/flow-ispo/pull/17#discussion_r1117894306